Sources/Sentry/Public/SentryOptions.h

```diff
@@ -120,6 +120,11 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableOutOfMemoryTracking;
 
+/**
+ * Whether to enable measuring app start or not. Default is NO.
+ */
+@property (nonatomic, assign) BOOL enableAppStartMeasuring;
```

Comment:
```suggestion
@property (nonatomic, assign) BOOL enableAppStartMeasurement;
```

not sure if any better, @bruno-garcia ?

Reply:
`enableAppStartMeasuring` is more aligned with  `enableOutOfMemoryTracking`, `enableAutoSessionTracking`. That's why I picked it.